### PR TITLE
Create docker images from sawtooth debs (volldockercontainerisierung...)

### PR DESCRIPTION
**Pull request by aludvik**
If you do `build_all installed` this will build docker images with the debs installed.

If you do `docker-compose -f docker/compose/sawtooth-demo.yaml up` in an environment running docker, docker will download the images from dockerhub and startup a running validator with tp_intkey_python, tp_xo_python, tp_config, tp_validator_registry, and the rest_api attached. The rest_api container will also have port 8080 exposed on the localhost.

### CODE REVIEW

File: docker/sawtooth-dev-tp_config [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2016 Intel Corporation
```
Comment (peterschwarz) [inline docker/sawtooth-dev-tp_config]: 2017

File: docker/sawtooth-dev-tp_intkey_python [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2016 Intel Corporation
```
Comment (peterschwarz) [inline docker/sawtooth-dev-tp_intkey_python]: 2017

File: docker/sawtooth-dev-validator [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Copyright 2016 Intel Corporation
```
Comment (peterschwarz) [inline docker/sawtooth-dev-validator]: 2017

**Comment by aludvik** [comment]
Fixed Copyright headers (again...)

File: docker/sawtooth-rest_api [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+ && apt-get update \
+ && apt-get install -y -q \
+    python3-sawtooth-rest-api \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+EXPOSE 40000/tcp
+EXPOSE 8080
+
+CMD ["rest_api"]
```
Comment (aludvik) [inline docker/sawtooth-rest_api]: This is wrong.

**Pull request merged**